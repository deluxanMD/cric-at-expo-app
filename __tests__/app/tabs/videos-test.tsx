import { render } from '@testing-library/react-native'

import VideosScreen from '@tabs/videos'

describe('<VideosScreen />', () => {
  test('VideosScreen renders correctly', () => {
    const tree = render(<VideosScreen />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
