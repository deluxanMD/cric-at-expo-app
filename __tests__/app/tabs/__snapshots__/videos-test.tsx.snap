// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VideosScreen /> VideosScreen renders correctly 1`] = `
<View>
  <Text>
    Videos
  </Text>
</View>
`;
